webapp/client/src/pages/RegistrationPage.js

```diff
@@ -33,7 +33,16 @@ export default function RegistrationPage({
   return (
     <>
       <StepHeaderButtons />
-      <FormHeader {...stepHeader} />
+      <FormHeader
+        {...stepHeader}
+        intro={
+          <Trans
+            t={t}
+            i18nKey="unlockCodeRequest.deadlineIntro"
```

Comment:
Can you please call this `unlockCodeRequest.input.intro` and relocate the message in the translations? - We're trying to keep the identifiers as close to the babel ones as possible. (in this case: `form.unlock-code-request.input-intro`)

Reply:
Fixed that

Reply:
@frederike-ramin I know I made that rule, but I'm doubting it now.

Originally, I didn't want to change too much in one go and wanted to make sure we can still search for the same key and find translations in both files while the React changes were still in a long-lived branch that hadn't been merged, so this was important because someone might have changed a text on main and we'd need to be able to port it to the React branches.

Now however, our PRs are smaller and we're making more atomic changes, so I think we can start organising translations in JS in a way _that makes sense_ to us rather than how they were in Babel.

WDYT?

Reply:
Completely on your side there. 👌